hdc/algo/ops/stats.py

```diff
@@ -232,7 +232,7 @@ def gammastd_yxt(
             s = gammastd(xt, nodata, cal_start, cal_stop)
             if (s != nodata).sum() > 0:
                 s = s * 1000
-                np.round_(s, 0, s)
+                np.round(s, 0, s)
```

Comment:
is that equivalent? I don't remember why I was going with `round_`, but I think it was memory related

Reply:
in numpy 1.26 it is:

```python
return around(a, decimals=decimals, out=out)
```

and then `around` and `round`, both are

```python
return _wrapfunc(a, 'round', decimals=decimals, out=out)
return _wrapfunc(a, 'round', decimals=decimals, out=out)
```